ignite/contrib/engines/common.py

```diff
@@ -190,7 +190,7 @@ def _setup_common_training_handlers(
 
     if with_gpu_stats:
         GpuInfo().attach(
-            trainer, name="gpu", event_name=Events.ITERATION_COMPLETED(every=log_every_iters)  # type: ignore[arg-type]
+            trainer, name="gpu", event_name=Events.ITERATION_COMPLETED(every=log_every_iters)  # type: ignore
```

Comment:
Changed this as it was giving an error in `flake8`
The exact error was
```
ignite/contrib/engines/common.py:193:96: F821 undefined name 'ignore'
ignite/contrib/engines/common.py:193:96: F821 undefined name 'arg'
```
The possible solutions I found were
- Change `ignore[arg-type]` to a general `ignore` (easiest to implement, but may be prone to errors)
- Similar [Issue](https://github.com/PyCQA/pyflakes/issues/475) , [solution](https://github.com/PyCQA/pyflakes/pull/455)  (Seems okay)
- Only use the allowed argument type (not sure if it is feasible)

Reply:
I'm not sure to understand what happens here and why it starts to fail in the unrelated place ?
If there is an issue, then master CI should fail as well, right ?

Reply:
Yes I was thinking the same thing. However, if I do keep it there, `pre-commit` gives me a `flake8` error mentioned above.

Reply:
Do you use the same flake8 version as on the CI ?

Reply:
Yes I think so
```bash
$ flak8 --version
3.8.4 (mccabe: 0.6.1, pycodestyle: 2.6.0, pyflakes: 2.2.0) CPython 3.8.5 on Linux
```
And by looking at the logs, the CI is also using the same version for all of them

Reply:
I'll revert this change and see if it fails here.